### PR TITLE
removed all legacy conceptId stuff

### PR DESCRIPTION
**Pull request by wheresrhys**

### CODE REVIEW

File: server/controllers/v1-v2-taxonomy.js [hunk 1]
```diff
@@ -34,7 +34,6 @@ module.exports = function(req, res, next) {
 				res.sendStatus(404);
 				return;
 			}
-			var tmeId = concordance.identifier.identifierValue;
```
Comment (matthew-andrews) [inline server/controllers/v1-v2-taxonomy.js]: i suspect i'm using tmeId for somehting else in this endpoint

Comment (wheresrhys) [inline server/controllers/v1-v2-taxonomy.js]: how strong is the suspicion?

Comment (matthew-andrews) [inline server/controllers/v1-v2-taxonomy.js]: https://github.com/Financial-Times/next-dobi/blob/conceptid-tidyup/server/controllers/v1-v2-taxonomy.js#L66

https://github.com/Financial-Times/next-dobi/blob/conceptid-tidyup/server/controllers/v1-v2-taxonomy.js#L98

Comment (matthew-andrews) [inline server/controllers/v1-v2-taxonomy.js]: though they could switch to `tag.term.id`

Comment (matthew-andrews) [inline server/controllers/v1-v2-taxonomy.js]: well, the second one could the first one cna't

Comment (wheresrhys) [inline server/controllers/v1-v2-taxonomy.js]: ah

**Comment by next-team** [comment]
I've spotted a [difference](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/dobi/2015-07-01/14:47-2088/failures/index.html) between your pull request and production — [see full output](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/dobi/2015-07-01/14:47-2088/successes/index.html)

**Comment by matthew-andrews** [comment]
:+1: ty

**Pull request merged**